src/array_partition.jl

```diff
@@ -72,50 +70,32 @@ Base.ones(A::ArrayPartition, dims::NTuple{N,Int}) where {N} = ones(A)
 
 for op in (:+, :-)
```

Comment:
Base got rid of all of these: do you think it is better to just delete these methods?

Reply:
Yes absolutely.